opentelemetry-sdk/benches/metric_counter.rs

```diff
@@ -30,12 +53,11 @@ fn counter_add(c: &mut Criterion) {
     let counter = create_counter();
     c.bench_function("Counter_Add_Sorted", |b| {
         b.iter(|| {
-            let mut rng = SmallRng::from_entropy();
             // 4*4*10*10 = 1600 time series.
-            let index_first_attribute = rng.gen_range(0..4);
-            let index_second_attribute = rng.gen_range(0..4);
-            let index_third_attribute = rng.gen_range(0..10);
-            let index_forth_attribute = rng.gen_range(0..10);
+            let index_first_attribute = CURRENT_RNG.with(|rng| rng.borrow_mut().gen_range(0..4));
```

Comment:
Can we generate four arrays filled with random numbers before we start? In each iteration, we assign an iteration index, then iterate over the pre-generated random numbers. This way, we separate the random number generation from the measurement.

Reply:
Good idea! I'll explore that next. At this point, we got the random to be manageable, so merging this PR to unblock progress.

Reply:
https://github.com/open-telemetry/opentelemetry-rust/issues/1805